web/util/queries.ts

```diff
@@ -49,6 +47,38 @@ const TICKET = `
   included,
   type,
 `;
+const SPEAKER = `
+  bio,
+  company,
+  name,
+  photo { ${FIGURE} },
+  pronouns,
+  slug,
+  social { twitter, linkedin },
+  title,
+  _createdAt,
+  _id,
+  _rev,
+  _type,
+  _updatedAt,
+  "sessions": *[_type == "session" && references(^._id) && !(_id in path("drafts.**"))] {
+    _id,
+    title,
+    duration,
+    "programContainingSession": *[_type == "program" && references(^._id)][0] {
```

Comment:
Nit: as discussed in Slack I'm not sure the default order is stable, so just in case this matches multiple programs it might be a good idea to use some explicit sort order (e.g. `_createdAt`)

Reply:
Good idea. edd5688.